src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -432,11 +442,31 @@ const analyticsEvents = {
       action: 'click',
       event: 'int-search-input-click',
       prefix: 'search-input',
+      ga4: {
+        event: 'interaction',
+        component_name: 'va-search-input',
+        custom_string_1: 'component-library',
+        /* Component to GA4 parameters */
+        mapping: {
+          'search-input-value': 'custom_string_2',
```

Comment:
What does `custom_string_2` refer to?

Reply:
Good question. That is one of the GA4 data layer parameters we have available to map to an event value. The `custom_string_1` will always be mapped to the value "component-library" and if we have any other dynamic values that we want to capture as part of the component event, we can use the other parameters on the list below. In this case, we are using `custom_string_2` to capture the dynamic search input value.

Do you think it would make more sense to make the GA4 key `value` instead of `custom_string_2`?

![Screenshot 2023-06-08 at 10 04 29 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/51197020-8806-4ff4-9298-9e39cb405a9c)


https://docs.google.com/spreadsheets/d/1c3Fd0f6rA0yRBeYuZEZzxzeLvJJclFUB5lX8hYPn7SY/edit#gid=2089299676

Reply:
@ataker Do you think it would make more sense to make the GA4 key `value` instead of `custom_string_2` in this case?

Reply:
I'm not super familiar with GA4 mapping, but if we're capturing the search value i think `value` makes sense.